hw/top_earlgrey/data/top_earlgrey.hjson

```diff
@@ -180,7 +180,17 @@
   //  the field and value should be module_inst.signal_name
   //  e.g flash_ctrl0.flash: [flash_phy0.flash_ctrl]
   inter_module: {
-    'flash_ctrl.flash': ['eflash.flash_ctrl']
+    'connect': {
+      'flash_ctrl.flash': ['eflash.flash_ctrl']
+    }
+
+    // top is to connect to top net/struct.
+    // It defines the signal in the top and connect from the module,
+    // use of the signal is up to top template
+    'top': [],
```

Comment:
@tjaychen Tim, I thought this can be used to connect reset manager output.

Reply:
sounds excellent.